src/pyos_sphinx_theme/__init__.py

```diff
@@ -112,6 +119,7 @@ def setup(app):
     app.add_html_theme("pyos_sphinx_theme", THEME_PATH)
     app.config.html_favicon = "https://www.pyopensci.org/images/favicon.ico"
     app.connect("builder-inited", update_config)
+    app.config.templates_path.append(str(TEMPLATE_PATH))
     # app.connect("html-page-context", redirect_from_html_to_dirhtml)
```

Comment:
Will the matomo analytics and css files be added later?  or is this  old code?

Reply:
> Will the matomo analytics and css files be added later? or is this old code?

no i didn't added it yet. Since i am unfamiliar with that script, i thought, we will do the first release and slowly migrate all those, addition js and css and do patch release in a regular manner if its ok for you.

Reply:
@Revathyvenugopal162 i'm sorry i am so slow to reply here. yes let's move forward with a release! we can add that later.